common/controlplaneconfig.h

```diff
@@ -141,6 +141,32 @@ class interface_t
 	common::globalBase::tFlow flow;
 };
 
+class bird_import_t
+{
+public:
+	void pop(common::stream_in_t& stream)
+	{
+		stream.pop(socket);
+		stream.pop(vrf);
+		stream.pop(flow);
+	}
+
+	void push(common::stream_out_t& stream) const
+	{
+		stream.push(socket);
+		stream.push(vrf);
+		stream.push(flow);
+	}
+
```

Comment:
Please rebase your branch onto the latest `main`.
There __will__ be conflicts because, for example, we don't use `push/pop` boilerplate anymore and instead do this:
```cpp
class config_t
{
public:
	config_t() = default;

	SERIALIZABLE(nat64stateful_id, dscp_mark_type, dscp, ipv6_prefixes, ipv4_prefixes, announces, next_module);

public:
	nat64stateful_id_t nat64stateful_id;
	common::eDscpMarkType dscp_mark_type{common::eDscpMarkType::never};
	uint8_t dscp{};
	std::vector<common::ipv6_prefix_t> ipv6_prefixes;
	std::vector<common::ipv4_prefix_t> ipv4_prefixes;
	std::set<common::ip_prefix_t> announces;
	controlplane::state_timeout state_timeout;
	std::string next_module;
	common::globalBase::flow_t flow;
};
```
That's just an example of the existing code. Note how we use the `SERIALIZABLE` macro with the names of the fields used for serialization

Reply:
Updated bird-import-proto and this branches.